Gemfile.lock

```diff
@@ -156,10 +157,8 @@ GEM
     em-socksify (0.3.1)
       eventmachine (>= 1.0.0.beta.4)
     erubis (2.7.0)
-    eventmachine (1.2.0.1)
-    eventmachine (1.2.0.1-x64-mingw32)
-    eventmachine (1.2.0.1-x86-mingw32)
-    execjs (2.6.0)
```

Comment:
@javan I see some downgrades, are they intentional from blade? or dropping of coffee-script versions, etc.